fh-messaging/lib/fhsrv.js

```diff
@@ -201,49 +203,62 @@ var msg = (function() {
       });
       return;
     }
-    //end request
-    res.json({"message":"accepted messages"});
 
     // first filter out any messages we already have
+    console.log("-----messages: ", messages);
     messaging.filterExistingMessages(topic, messages, function(err, existingMessages, newMessages) {
       var timings_filtered = Date.now();
       if (err) {
         logger.error("Error filtering after " + (timings_filtered - timings_start) + " milliseconds", err);
         return;
       }
-      if (newMessages.length > 0) {
-        messaging.logMessage(topic, newMessages, function(err, objects) {
-          var i;
-          if (err) {
-            logger.error("Error filtering after " + (timings_filtered - timings_start) + " milliseconds", err);
-            return;
-          }
 
-          var ids = [];
-          for (i in objects) {
-            if (objects.hasOwnProperty(i)) {
-              var msg = objects[i];
-              ids.push(msg.MD5);
-            }
-          }
-          var m = {};
-          m.newMessages = ids;
-          if (existingMessages.length > 0) {
-            m.existingMessages = existingMessages;
+      console.log("------existingMessages: ", existingMessages);
+      console.log("-------newMessages: ", newMessages);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)